### PR TITLE
Install Rubocop

### PR DESCRIPTION
**Pull request by oscarjosecostero**
ref #6 

Rubocop Installed and ready.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -29,7 +28,7 @@ gem 'jbuilder', '~> 2.5'
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'rubocop', require: false
```
Comment (gbrlmrllo) [inline Gemfile]: This gem should go into the development group.

https://github.com/oscarjosecostero/Easyrent/blob/ec2b4aecf5f5d5f82a8f99609b8cef2959422c23/Gemfile#L44

**Review by gbrlmrllo — CHANGES_REQUESTED** [review]
Good job!, only one comment.

File: Gemfile (hunk 1 above)
Comment (oscarjosecostero) [reply Gemfile]: You are right.
Should i correct all of the Rubocop ofenses?.
There are several that we can modify. Like "Line too long" (should it be allways 80?) or "missing top-level class documentation comment", in that case, do i modify them? or should we stick to rubocop's suggestions strictly?

Comment (gbrlmrllo) [reply Gemfile]: `Should i correct all of the Rubocop ofenses?.`

No, we'll do it on the way.

`There are several that we can modify. Like "Line too long" (should it be allways 80?) or "missing top-level class documentation comment", in that case, do i modify them? or should we stick to rubocop's suggestions strictly?`

Not all of us should strictly follow them, we could come to an agreement later.

File: Gemfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 git_source(:github) do |repo_name|
-  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  repo_name = '#{repo_name}/#{repo_name}' unless repo_name.include?('/')
```
Comment (gbrlmrllo) [inline Gemfile]: All variables interpolation must be in double quotation marks.
 
Eg.

`"#{repo_name}/#{repo_name}"`

File: Gemfile [hunk 3]
```diff
@@ -28,27 +28,27 @@ gem 'jbuilder', '~> 2.5'
...
   gem 'capybara', '~> 2.13'
   gem 'selenium-webdriver'
 end
 
 group :development do
-  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
-  gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'rubocop', require: false
```
Comment (gbrlmrllo) [inline Gemfile:45]: 👍

File: Gemfile [hunk 4]
```diff
@@ -28,27 +28,27 @@ gem 'jbuilder', '~> 2.5'
...
+  # Spring speeds up development by keeping your application running in the background.
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
+  gem 'web-console', '>= 3.3.0'
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
```
Comment (gbrlmrllo) [inline Gemfile:54]: 👍

File: Rakefile [hunk 5]
```diff
@@ -1,5 +1,6 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
-# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
```
Comment (gbrlmrllo) [inline Rakefile:2]: This is a case that we could overlook, total, are just comments.

File: app/mailers/user_notifier_mailer.rb [hunk 6]
```diff
@@ -1,11 +1,13 @@
...
+  default from: 'Easyrent@mail.com'
 
-  # send a signup email to the user, pass in the user object that contains the user's email address
+  # send a signup email to the user, pass in the user
+  # object that contains the user's email address
   def send_signup_email(user)
     @user = user
-    mail( :to => @user.email,
-    :subject => 'Thanks for signing up for our amazing app' )
+    mail(
```
Comment (gbrlmrllo) [inline app/mailers/user_notifier_mailer.rb:8]: 👍

**Review by gbrlmrllo — CHANGES_REQUESTED** [review]
Cool, only two comment.

File: Rakefile (hunk 5 above)
Comment (oscarjosecostero) [reply Rakefile:2]: Exactly. I only did it in that part. After that i disabled that verification:

Metrics/LineLength:
  Enabled: false

Temporally, until we come to an agreement in that issue.

Comment (oscarjosecostero) [inline Rakefile]: Exactly. I only did it in that part. After that i disabled that verification:

Metrics/LineLength:
  Enabled: false

Temporally, until we come to an agreement in that issue.

Comment (gbrlmrllo) [reply Rakefile:2]: cool!

**Review by gbrlmrllo — APPROVED** [review]

**Pull request merged**